website/static/js/share/index.js

```diff
@@ -79,14 +86,21 @@ ShareApp.controller = function() {
     }).then(function(data) {
         self.vm.ProviderMap = data.providerMap;
 
-        self.sideBarController = new SideBar.controller(self.vm);
         self.statsController = new Stats.controller(self.vm);
-        self.resultsController = new Results.controller(self.vm);
         self.searchBarController = new SearchBar.controller(self.vm);
-        self.footerController = new Footer.controller(self.vm);
 
     });
 
+    m.request({
```

Comment:
Yes, I think it is only initialized once. A single request is made when searching (the actual search request) or manipulating other elements on the page.